styles.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: fuchsia;
+}
```

Comment:
- [ ] It is the best practice to use class selector rather than element selector. It is for semantics reason. Kindly use the class name from  your index file to style the content of the `h1`.